hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -446,6 +446,12 @@
               If not, the address size of the command is always 3B.
               '''
           }
+          { bits:   "11"
+            name:   "mbyte_en"
```

Comment:
sorry i can't remember, what's an `mbyte`?

Reply:
It is a byte follows address field existed in Dual IO/ Quad IO commands.